### PR TITLE
Mobile Version Skeleton

### PR DESCRIPTION
**Pull request by techEmprez**
In this repository, we:
- Added index.html file
- Added images and logo
- Added styles.css to file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <div class="headline">
+      <h1>Charlie Giorgania</h1>
+      <p class="description">
+        Hello! I am a software developer! I can help you build a product,<br />
+        feature or website. Taka a look of my works. If you like what you see<br />
+        and have a project you need coded, don’t hesitate and contact me.
+      </p>
+
+      <button class="despButton" type="submit">Start collaboration</button>
+    </div>
```
Comment (muneebulrehman) [inline index.html:24]: Let's wrap this section in a `section` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  </body>
+</html>
```
Comment (muneebulrehman) [inline index.html:26]: Your application is not matching the Figma template you choose. Kindly update it according to the Figma design. If you notice, your content is not aligned properly and also there is some white space on the right side. Kindly fix this also. Also, use the same margin and padding as in the Figma design.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

File: README.md [hunk 3]
```diff
@@ -1 +1,29 @@
-# My_Portfolio
\ No newline at end of file
+# My_Portfolio
+
+A simple "Portfolio" web page
```
[the remark this answers is not in the document]
Comment (Suigeneriz) [reply README.md:3]: I edited my portfolio desctiption.

File: index.html (hunk 2 above)
Comment (Suigeneriz) [reply index.html]: Fixed the styling with the figma file and added the remaining image for the "Mobile Version" of the portfolio

Comment (Suigeneriz) [reply index.html]: Fixed it as figma file replicate

File: index.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <nav class="navLogo">
+      <a href="./index.html">
+        <img src="./images/navlogo.png" alt="" />
+      </a>
+      <img class="navHam" src="./images/hamburger.png" alt="" />
```
Comment (tufoinnkuo10) [inline index.html:15]: - [optional ] please the whole heading section should be inside a `header tag`

Comment (tufoinnkuo10) [inline index.html:15]: - [ optional] only the hamburger should be inside the nav tag. the logo should be outside of it

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark: ✔️ 

Hi @Suigeneriz 

Your project is complete! Great job for making all the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️ 
- Validations correctly implemented✔️ 
- No linter errors✔️ 
- design well followed ✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**